pkg/tsdb/cloudwatch/get_dimension_values_for_wildcards_test.go

```diff
@@ -99,10 +64,135 @@ func TestGetDimensionValuesForWildcards(t *testing.T) {
 			{MetricName: utils.Pointer("Test_MetricName"), Dimensions: []*cloudwatch.Dimension{{Name: utils.Pointer("Test_DimensionName2"), Value: utils.Pointer("Value")}}},
 		}
 		api.On("ListMetricsPagesWithContext").Return(nil)
-		queries, err = executor.getDimensionValuesForWildcards(ctx, "us-east-1", api, []*models.CloudWatchQuery{query}, tagValueCache, 50)
+		queries, err = executor.getDimensionValuesForWildcards(ctx, "us-east-1", api, []*models.CloudWatchQuery{query}, tagValueCache, 50, noSkip)
 		assert.Nil(t, err)
 		assert.Len(t, queries, 1)
 		assert.Equal(t, map[string][]string{"Test_DimensionName2": {"Value"}}, queries[0].Dimensions)
 		api.AssertExpectations(t)
 	})
+
+	t.Run("MetricSearch query type", func(t *testing.T) {
```

Comment:
Is there a reason these moved?

Reply:
i was grouping these tests by the query type. The tests that were left in their original spots are for caching so i left them ungrouped.